.github/workflows/bench.yml

```diff
@@ -43,13 +43,20 @@ jobs:
       - name: Prepare machine
         run: sudo /root/bin/prep.sh
 
-      # Pin the benchmark to core 0 and run it at elevated priority.
+      # Pin the benchmark run to core 0 and run all benchmarks at elevated priority.
       - name: Benchmark
         run: |
           nice -n -20 taskset -c 0 \
             cargo +$TOOLCHAIN bench --features ci,bench | tee output.txt
 
-      # Enable turboboost, hyperthreading and use powersave governor.
+      # Pin the transfer benchmark to core 0 and run it at elevated priority inside perf.
+      - name: Perf transfer benchmark
+        run: |
+          nice -n -20 taskset -c 0 \
+            perf record -F997 --call-graph=lbr -o perf.data \
+            cargo +$TOOLCHAIN bench --features ci,bench --bench transfer
```

Comment:
You should add `--exact` when matching, just so that you don't pick up other stuff.

I recommend `cargo bench -F bench --bench transfer -- --exact "Run multiple transfers with varying seeds"`

Reply:
Ah, thanks! I didn't know one could match on that long string.